src/jsMain/kotlin/ui/components/tabs/entrytab/ManualEntryTab.kt

```diff
@@ -149,7 +153,7 @@ class ManualEntryTab : RComponent<ManualEntryTabProps, ManualEntryTabState>() {
                         errorMessage = "Never gonna give you up, never gonna let you down, never gonna run around..."
                         displayingError = true
                     } else {
-                        errorMessage = "Cannot parse entered text as valid JSON/XML."
+                        errorMessage = props.polyglot.t("manual_entry_exception")
```

Comment:
The key here should be something more specific, like `manual_entry_cannot_parse_exception`